File: lua/refactoring/tests/debug/print_var/go/multiple-statements/print_var.expected.go
package main

import "fmt"

func simple_function(a int) {
    var test int = 1
    test_other := 1
    // __AUTO_GENERATED_PRINT_VAR_START__
    fmt.Println(fmt.Sprintf("custom print var simple_function test_other:: %v", test_other)) // __AUTO_GENERATED_PRINT_VAR_END__
    for idx := test - 1; idx < test_other; idx++ {
        fmt.Println(idx, a)
    }
}

File: lua/refactoring/tests/debug/print_var/lua/dot-index-expression-table/print_var.expected.lua
local function simple_function()
    local test = {
        test_other = 11,
    }
    print(test.test_other)
    -- __AUTO_GENERATED_PRINT_VAR_START__
    print([==[simple_function test:]==], vim.inspect(test)) -- __AUTO_GENERATED_PRINT_VAR_END__
end
